CRM/Core/DAO.php

```diff
@@ -2398,7 +2399,26 @@ public static function appendCustomTablesExtendingContacts(&$cidRefs) {
     $customValueTables = CRM_Core_BAO_CustomGroup::getAllCustomGroupsByBaseEntity('Contact');
     $customValueTables->find();
     while ($customValueTables->fetch()) {
-      $cidRefs[$customValueTables->table_name] = ['entity_id'];
+      $cidRefs[$customValueTables->table_name][] = 'entity_id';
```

Comment:
NFC - just a bit more defensive as it doesn't muck things up if the call order is changed.